### ISSUE TITLE
Introduction of file system attribute

### ISSUE THREAD

**Issue by thompson-tomo** [open-telemetry/semantic-conventions#1108]
### Area(s)

area:file

### Is your change request related to a problem? Please describe.

I want to describe the underlying provider storing my files

### Describe the solution you'd like

I would like the introduction of an additional attribute "system" which describes the system storing the files just like we have for databases & messaging. The strongly typed values would be:

- LogicalDisk
- AwsS3
- AzureBlob
- ...

**Comment by joaopgrassi** [comment]
See https://github.com/open-telemetry/semantic-conventions/pull/914. CC @trisch-me

**Comment by thompson-tomo** [comment]
@joaopgrassi yeap I had seen that PR but as far as I can tell there is no strongly typed field which could be used for the purpose mentioned here.

**Comment by trisch-me** [comment]
Yes we don't have there such a field. You can propose it in separate PR
